types/resource.go

```diff
@@ -154,9 +154,10 @@ type EngineSpec struct {
 
 type EngineStatus struct {
 	InstanceStatus
-	ReplicaModeMap     map[string]ReplicaMode `json:"replicaModeMap"`
-	Endpoint           string                 `json:"endpoint"`
-	LastRestoredBackup string                 `json:"lastRestoredBackup"`
+	ReplicaModeMap     map[string]ReplicaMode       `json:"replicaModeMap"`
+	Endpoint           string                       `json:"endpoint"`
+	LastRestoredBackup string                       `json:"lastRestoredBackup"`
+	BackupList         map[string]*BackupStatusInfo `json:"backups"`
```

Comment:
Keep the name consistent.

`BackupStatus map[string]*BackupStatus json:"backupStatus"`